typespec-extension/changelog.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 0.8.0 (Unreleased)
```

Comment:
we still uses this version range `>=0.45.0 <1.0.0"`.

not sure when Laurent's proposal get consensus.

hence probably delay the release a few days (unless asked for).